### DISCUSSION TITLE
How can we achieve HA for both Age and Postgres DB?

### DISCUSSION THREAD

**Discussion by ezeeyahoo** [apache/age#1201] [Q&A]
Hello Team,

I have been searching for a guide or so to deploy multiple instances of Age (Highly available Age) connected with multiple instances of Postgres DB (Highly available Postgres) that eventually connected to persistent storage. Please correct my question if it does not fit logically.

Can somebody help me, to find relevant documentation?

**Comment by eyab** [comment]
Hello Eshnat,

It is nice to meet you. My name is Eya, and I am Apache AGE chair. Thank
you for your message and for reaching out. Please give us more information
regarding your usecase; we will gladly support you.

You can also reach out to me at ***@***.*** to talk about this more.

Best regards,

**Comment by ezeeyahoo** [comment]
Nice to meet you too Eya.

So the background is We have an OCP environment where we want highly available AGE and highly available Postgres DB so that service downtime is negligible. We have a REST API in the same environment, deployed with replicas per tenant(graph), so that no tenant block any other tenant operations. In such a paradigm, we need a highly available age and highly available Postgres service at the backend so that disruption in service for one tenant does not block access to persistent storage for others.

**Comment by ezeeyahoo** [comment]
Anybody?
